src/transforms/transform-pull-request.ts

```diff
@@ -87,40 +85,57 @@ export const transformPullRequest = async (
 	log?: Logger
 ) =>
 {
-	const { head } = pullRequest;
+	const {
+		id,
+		user,
+		comments,
+		base,
+		number,
```

Comment:
/nit
`number` looks a bit confusing. Could be `pullRequestNumber` <-- But that draw the question, is this `destructing` really help the readability?
For me when I saw the code bellow, i am confused.
```javascript
pullRequests: [
			{
				...
				id: number,
        ...
      }
]
```
I'd rather see
```javascript
pullRequests: [
			{
				...
				id: pullRequest.number,
        ...
      }
]
```

Reply:
But `/nit`, I understand there's a code convention and consistent sake. So just random thing at 5pm Wed night. Nothing super important here.